Dockerfile.test.rhel7

```diff
@@ -0,0 +1,63 @@
+FROM registry.access.redhat.com/ubi7/ubi
+
+ENV INSTALL_DIR=/opt/conjur-api-python3
+
+# Copy below repo file to provide missing rpms
+COPY ./test/RPM-GPG-KEY-CentOS-7 /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+COPY ./test/CentOS-SCLo-scl-rh.repo /etc/yum.repos.d/CentOS-SCLo-scl-rh.repo
+COPY ./test/CentOS-7-Linux-AppStream.repo /etc/yum.repos.d/CentOS-Linux-AppStream.repo
+RUN gpg --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+
+# enable repos
+RUN yum-config-manager --enable ubi-server-rhscl-7-rpms
+RUN yum-config-manager --enable ubi-7-server-optional-rpms
+
+RUN yum --disableplugin=subscription-manager \
+                                install -y  bash \
+                                            wget \
+                                            binutils \
+                                            yum-utils \
+                                            gcc gcc-c++ make \
+                                            curl \
+                                            jq \
+                                            libffi-devel \
+                                            gnome-keyring \
+                                            procps \
+                                            zlib-devel \
+                                            wget \
+                                            openssl-devel \
+                                            bzip2-devel \
+                                            dbus-x11 \
+                                            && yum --disableplugin=subscription-manager clean all
+
+
+# install python 3.9
+ENV PYTHON_CONFIGURE_OPTS="--enable-shared"
+RUN wget https://www.python.org/ftp/python/3.9.6/Python-3.9.6.tgz
+RUN tar xzf Python-3.9.6.tgz
+RUN ./Python-3.9.6/configure --enable-shared
+RUN make altinstall
+
+# configure dbus-11
+RUN dbus-uuidgen > /var/lib/dbus/machine-id
+
+RUN mkdir -p $INSTALL_DIR
+WORKDIR $INSTALL_DIR
+COPY ./requirements.txt $INSTALL_DIR/
+RUN yum --disableplugin=subscription-manager install -y rh-python38-python-devel \
+                                                        centos-release-scl \
+                                                        devtoolset-7 \
+                                                        python3-pip
+RUN scl enable devtoolset-7 bash
+RUN cp /usr/local/bin/python3.9 /usr/local/bin/python3
+ENV PATH=/usr/local/bin:${PATH}
+RUN ldconfig /usr/local/lib
+ENV PATH=${PATH}:/usr/local/lib
+RUN pip3.9 install wheel
+RUN pip3.9 install -r requirements.txt
+COPY ./bin/build_integrations_tests_runner /build_integrations_tests_runner
+COPY . $INSTALL_DIR
+COPY ./test/configure_test_executor.sh /configure_test_executor.sh
```

Comment:
Pleas work to separate the COPYs RUNs etc by a new line. Otherwise it’s hard to digest what is happening here